platform/android/SCsub

```diff
@@ -169,3 +169,15 @@ if lib_arch_dir != '':
     if ndk_version != None and LooseVersion(ndk_version) >= LooseVersion("15.0.4075724"):
         stl_lib_path = str(env['ANDROID_NDK_ROOT']) + '/sources/cxx-stl/llvm-libc++/libs/' + lib_arch_dir + '/libc++_shared.so'
         env_android.Command(out_dir + '/libc++_shared.so', stl_lib_path, Copy("$TARGET", "$SOURCE"))
+
+    if 'module_arcore_enabled' in env and env['module_arcore_enabled']:
```

Comment:
Need to see if we can move this into modules\arcore\SCSub somehow

Reply:
Fixed in BastiaanOlij/godot#9.